.goreleaser.yml

```diff
@@ -2,14 +2,6 @@ dist: goreleaser
 
 project_name: pulumi
 
-blobs:
-- bucket: get.pulumi.com
-  folder: releases/sdk/
-  ids:
-    - pulumi
-  provider: s3
-  region: us-west-2
-
```

Comment:
No longer used, replaced by an aws cli action in release.yml: 

https://github.com/pulumi/pulumi/blob/master/.github/workflows/release.yml#L112